_posts/2021/2021-01-21-why-i-love-working-as-an-applicaton-management-engineer-at-epages.md

```diff
@@ -0,0 +1,75 @@
+---
+layout: post
+title: What working as an Application Management Engineer is all about
```

Comment:
Could you check back with Aldijana or Sarah if that works out? Somehow the post wasn't really about why he loves the job... That's why I came up with this